### PR TITLE
Fix register page alertSpy bug

### LINKED ISSUE #146: The alert.present element is not in register page AlertSpy
The `present` method is not included in the AlertSpy object causing a late security error.

Copy of error:
```
HeadlessChrome 85.0.4183 (Linux 0.0.0) RegisterPage should alert the user when opening the page FAILED
	Uncaught Error: Uncaught (in promise): TypeError: Cannot read property 'present' of undefined
	TypeError: Cannot read property 'present' of undefined
	    at RegisterPage.<anonymous> (http://localhost:9876/_karma_webpack_/src/app/login/register/register.page.ts:41:17)
	    at step (http://localhost:9876/_karma_webpack_/node_modules/tslib/tslib.es6.js:100:1)
	    at Object.next (http://localhost:9876/_karma_webpack_/node_modules/tslib/tslib.es6.js:81:45)
	    at fulfilled (http://localhost:9876/_karma_webpack_/node_modules/tslib/tslib.es6.js:71:42)
	    at ZoneDelegate../node_modules/zone.js/dist/zone.js.ZoneDelegate.invoke (http://localhost:9876/_karma_webpack_/node_modules/zone.js/dist/zone.js:386:1)
	    at ProxyZoneSpec.push../node_modules/zone.js/dist/zone-testing.js.ProxyZoneSpec.onInvoke (http://localhost:9876/_karma_webpack_/node_modules/zone.js/dist/zone-testing.js:292:1)
	    at ZoneDelegate../node_modules/zone.js/dist/zone.js.ZoneDelegate.invoke (http://localhost:9876/_karma_webpack_/node_modules/zone.js/dist/zone.js:385:1)
	    at Object.onInvoke (http://localhost:9876/_karma_webpack_/node_modules/@angular/core/fesm5/core.js:26256:1)
	    at ZoneDelegate../node_modules/zone.js/dist/zone.js.ZoneDelegate.invoke (http://localhost:9876/_karma_webpack_/node_modules/zone.js/dist/zone.js:385:1)
	    at Zone../node_modules/zone.js/dist/zone.js.Zone.run (http://localhost:9876/_karma_webpack_/node_modules/zone.js/dist/zone.js:143:1) thrown
```

### PR DESCRIPTION
**Pull request by marickmanrho**
Adds:

- A present() method to the jasmine spyObj of the AlertController.

Fixes: #146

### CODE REVIEW

**Comment by marickmanrho** [comment]
Since this was an inconsistent error, I've run the test at least 30 times. All passed.

**Comment by HendrikPreuter** [comment]
I ran the tests ten times myself and all passed.

File: gridt/src/app/login/register/register.page.spec.ts [hunk 1]
```diff
@@ -11,7 +11,15 @@ describe("RegisterPage", () => {
   //testing suite, (de pagina te testen)
```
Comment (HendrikPreuter) [inline gridt/src/app/login/register/register.page.spec.ts]: Can you remove my comments? They don't fit and should have been removed by me before pushing.

File: gridt/src/app/login/register/register.page.spec.ts [hunk 2]
```diff
@@ -11,7 +11,15 @@ describe("RegisterPage", () => {
...
+  let alertSpy: AlertController = jasmine.createSpyObj("alertSpy", {
+    "create": {
+      "present": function() {
+        return true
+      }},
+    "dismiss": function() {
+      return true
+    }}
+  );
 //worden elke test opnieuw geladen
```
Comment (HendrikPreuter) [inline gridt/src/app/login/register/register.page.spec.ts]: same here.

File: gridt/src/app/login/register/register.page.spec.ts [hunk 3]
```diff
@@ -11,7 +11,15 @@ describe("RegisterPage", () => {
...
   let component: RegisterPage;
   let fixture: ComponentFixture<RegisterPage>;
-  let alertSpy: AlertController = jasmine.createSpyObj("alertSpy", ["create", "dismiss"]);
+  let alertSpy: AlertController = jasmine.createSpyObj("alertSpy", {
+    "create": {
+      "present": function() {
+        return true
+      }},
+    "dismiss": function() {
+      return true
```
Comment (HendrikPreuter) [inline gridt/src/app/login/register/register.page.spec.ts:19]: This seems to solve the issue for me as i ran the tests ten times and none of them failed.

**Review by HendrikPreuter — APPROVED** [review]

**Review by Drvanon — APPROVED** [review]

**Pull request merged**